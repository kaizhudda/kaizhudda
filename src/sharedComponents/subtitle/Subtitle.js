import React from 'react';

import './Subtitle.scss';

const Subtitle = ({ children }) => {
  return (
    <h3 className="Subtitle">
      {children}
    </h3>
  );
}

export default Subtitle;
