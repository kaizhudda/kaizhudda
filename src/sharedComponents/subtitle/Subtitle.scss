@import '../../styles/variables';
@import '../../styles/media-queries';

.Subtitle {
  font-weight: 300;
  font-size: 1.375rem;
  margin-top: 0px;

  @include phone-only {
    font-size: 1.125rem;
  }
}