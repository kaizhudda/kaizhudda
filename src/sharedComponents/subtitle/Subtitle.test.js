import React from 'react';
import { shallow } from 'enzyme';
import Subtitle from './index';

const title = 'Test Title';
let wrapped = shallow(<Subtitle>{title}</Subtitle>); 

describe('Title', () => {
  it('should render the Subtitle Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('renders the Subtitles children', () => {
    expect(wrapped.find('h3').text()).toEqual(title);
  });
});
