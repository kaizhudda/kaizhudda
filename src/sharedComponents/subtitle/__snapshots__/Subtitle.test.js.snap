// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Subtitle Component correctly 1`] = `
<h3
  className="Subtitle"
>
  Test Title
</h3>
`;
